test-apps/display-test-app/android/imodeljs-test-app/app/build.gradle

```diff
@@ -1,17 +1,19 @@
-apply plugin: 'com.android.application'
+plugins {
+    id 'com.android.application'
+}
 
 android {
-    compileSdkVersion 28
-    ndkVersion "18.1.5063045"
+    compileSdk 32
 
     defaultConfig {
         applicationId "com.bentley.imodeljs_test_app"
-        minSdkVersion 28
-        targetSdkVersion 28
+        minSdk 24
```

Comment:
why was this dropped from 28 -> 24?

Reply:
This matches what we have in our [mobile-sdk-android](https://github.com/iTwin/mobile-sdk-android/blob/main/mobile-sdk/build.gradle#L17).

Reply:
but why is the mobile-sdk-android using that??

if i'm understanding correctly [sdk 24](https://developer.android.com/studio/releases/platforms#7.0) is android 7.0, back from 2016. Why are we shooting ourselves in the foot by trying to support a version so old?

Reply:
We haven't done any official Android releases of the mobile-sdk yet so this is up for discussion. I don't think we've had any problems or restrictions supporting that minSdk so far.

For reference here are the distribution numbers according to Android Studio:
<img width="1167" alt="image" src="https://user-images.githubusercontent.com/48103957/199791534-a7f87509-55e3-46c5-b8a1-5ec0c86ef737.png">

Reply:
in my naive opinion, we should probably be targeting 28 or 29+

Reply:
We should ask for input from SYNCHRO, but I don't have any problem updating the value. Please note that Todd's change here updated DTA to match the add-on (as well as itwin/mobile-sdk-android, which also is configured to match the add-on). So if we want to update the required version, we need to make sure to also update the add-on.

Reply:
Created https://github.com/iTwin/itwinjs-backlog/issues/504, we already had https://github.com/iTwin/itwinjs-backlog/issues/372

Reply:
let's follow up on this with stakeholders and see if we can update. Last time we discussed (last year) it needed to stay at 24 but no idea what that looks like now.